src/ResourceManager/Storage/Commands.Management.Storage.Test/ScenarioTests/StorageDataPlaneTests.ps1

```diff
@@ -375,14 +376,44 @@ function Test-Common
         $LoggingOperations = "All"
 
         Set-AzStorageServiceLoggingProperty -ServiceType blob -RetentionDays $retentionDays -Version $version -LoggingOperations $LoggingOperations -Context $storageContext
-        $property = Get-AzStorageServiceLoggingProperty -ServiceType blob -Context $storageContext
-        Assert-AreEqual $LoggingOperations $property.LoggingOperations.ToString()  #INVESTIGATE, sometimes fails
+        $i = 0
```

Comment:
One Minor thing, can we set the properties together, and run the wait function to wait for the change updated in server togehther. This will make the function looks more clear and easy to maintain, and make the test runs quicker.

Anyway, I am also good if you would like to keep current code.

Reply:
I will add this suggestion to a separate issue along with the improvement where a single storage account and resource group is used for all the tests.

Reply:
Issue #7162